libs/ledger/src/upow/synergetic_contract.cpp

```diff
@@ -330,8 +288,7 @@ Status SynergeticContract::Complete(uint64_t block, BitVector const &shards)
 
   // setup the storage infrastructure
   CachedStorageAdapter storage_cache(*storage_);
-  StateSentinelAdapter state_sentinel{
-      storage_cache, Identifier{digest_.ToHex() + "." + std::to_string(block)}, shards};
+  StateSentinelAdapter state_sentinel{storage_cache, Identifier{digest_.ToHex()}, shards};
```

Comment:
Maybe I am wrong but this presumable sets the state space prefix as:

`<digest>.state.`

Where as we want:

`<digest>.<address>.state.`